tests/test_input.cpp

```diff
@@ -0,0 +1,41 @@
+#include <catch2/catch.hpp>
+
+#include <faasm/emulator.h>
+
+#include "faabric/util/func.h"
+#include "faasm/input.h"
+
+namespace tests {
+TEST_CASE("Test parse string to ints", "[input]")
+{
+    const char* input = "0 123 99987 5";
```

Comment:
Do we need to `cleanCppTests` here as well?

Reply:
No, this is just doing string parsing so doesn't interact with anything Faabric- or emulator-related